src/WebJobs.Script/Binding/ServiceBus/ServiceBusValidator.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.Script.Extensibility;
+using Microsoft.ServiceBus;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public interface INamespaceManager
+    {
+        void TestManageRights();
+    }
+
+    public class DefaultNamespaceManager: INamespaceManager
+    {
+        NamespaceManager _manager;
+        public DefaultNamespaceManager(NamespaceManager manager)
+        {
+            _manager = manager;
+        }
+
+        public void TestManageRights()
+        {
+            _manager.GetQueues();
+        }
+    }
+
+    public static class ServiceBusValidator
+    {
+        public static void ValidateAccessRights(string bindingName, INamespaceManager manager, TraceWriter traceWriter)
+        {
+            if (Utility.IsDynamic())
+            {
+                try
+                {
+                    // namespace manager apis require Manage rights
+                    manager.TestManageRights();
+                }
+                catch (UnauthorizedAccessException)
+                {
+                    traceWriter.Warning($"Service Bus Trigger binding '{bindingName}' requires a connection string with Manage AccessRights for correct scaling");
```

Comment:
Slight tweak to message: "... for correct triggering and scaling behavior when running in a consumption plan."